SPECS/docker-compose/docker-compose.spec

```
@@ -0,0 +1,56 @@
+Summary:        Define and run multi-container applications with Docker
+Name:           docker-compose
+Version:        2.16.0
+Release:        1%{?dist}
+License:        MIT
```

Looks like this part of docker is Apache-2.0
```suggestion
License:        Apache-2.0
```

---

I think there are two licenses at question
- The license of the package (source code)
- The license of the spec file (this one)
I think all the original spec that we author, we generally keep them MIT